pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -145,7 +145,10 @@ func computeScheduleRecurrence(
 	}
 	expr, err := cronexpr.Parse(cron)
 	if err != nil {
-		return nil, err
+		return nil, errors.Newf(
+			`error parsing schedule expression: %q.
+See https://en.wikipedia.org/wiki/Cron for more details.
```

Comment:
I don't think we need to link to the cron wikipedia page. I think people know what cron is. We could `must be valid cron syntax or valid option (see docs)` or something.

Reply:
Sure -- changed.